test/integration/cms/toolbar_test.rb

```diff
@@ -8,9 +8,12 @@ def setup
   end
 
   test 'CMS toolbar rendering' do
+    cms_token = @provider.settings.cms_token!
+    expires_at = Time.now.utc.round + 1.minute
```

Comment:
And another one in case you move normalization to the signature module
```suggestion
    expires_at = Time.now + 1.minute
```